slycot/src/AB08NX.f

```diff
@@ -199,8 +199,8 @@ SUBROUTINE AB08NX( N, M, P, RO, SIGMA, SVLMAX, ABCD, LDABCD,
       INTEGER           ILAENV
       EXTERNAL          ILAENV
 C     .. External Subroutines ..
-      EXTERNAL          DLAPMT, DLARFG, DLASET, DLATZM, DORMQR, DORMRQ,
-     $                  MB03OY, MB03PY, XERBLA
+      EXTERNAL          DLAPMT, DLARFG, DLASET, SLCT_DLATZM, DORMQR,
```

Comment:
OK, fair enough.

FWIW, my [go-to reference for Fortran 77](http://www.star.le.ac.uk/~cgp/prof77.html#tth_sEc14) says "One of the most irksome restrictions of Fortran77 is that symbolic names cannot be more than six characters long."